snippets/fsharp/VS_Snippets_CLR_System/system.console.class.unsafe/fs/setfont1.fs

```diff
@@ -0,0 +1,80 @@
+﻿// <Snippet3>
+open System
+open System.Runtime.InteropServices
+
+[<LiteralAttribute>]
```

Comment:
This can be shortened to `[<Literal>]` - in VSCode the shortening isn't offered by the tooling, unfortunately, but in VS it is.

Reply:
Yes, I knew that one. I'm just used to it being auto corrected in C# for me for VS I guess. I'll make the change.